stdlib/public/Glibc/CMakeLists.txt

```diff
@@ -3,44 +3,57 @@ set(sources
     Misc.c
 )
 
-set(output_dir "${SWIFTLIB_DIR}/glibc")
+# When cross-compiling the stdlib on Unix platforms, we'll need a separate
+# glibc for each target.
+foreach(SDK ${SWIFT_SDKS})
+  foreach(arch ${SWIFT_SDK_${SDK}_ARCHITECTURES})
+    # Don't generate Glibc module maps for Darwin targets.
+    if("${SDK}" STREQUAL "LINUX" OR "${SDK}" STREQUAL "FREEBSD")
```

Comment:
This `if()` statement is the only difference between this and https://github.com/apple/swift/pull/1679. :point_up: